butterknife-gradle-plugin/src/test/resources/fixtures/R2.java

```diff
@@ -20,79 +20,79 @@
 public final class R2 {
   public static final class anim {
     @AnimRes
-    public static final int res = 0x7f040001;
+    public static final int res = 1;
   }
 
   public static final class array {
     @ArrayRes
-    public static final int res = 0x7f040002;
+    public static final int res = 2;
   }
 
   public static final class attr {
     @AttrRes
-    public static final int res = 0x7f040003;
+    public static final int res = 3;
   }
 
   public static final class bool {
     @BoolRes
-    public static final int res = 0x7f040004;
+    public static final int res = 4;
   }
 
   public static final class color {
     @ColorRes
-    public static final int res = 0x7f040005;
+    public static final int res = 5;
   }
 
   public static final class dimen {
     @DimenRes
-    public static final int res = 0x7f040006;
+    public static final int res = 6;
   }
 
   public static final class drawable {
     @DrawableRes
-    public static final int res = 0x7f040007;
+    public static final int res = 7;
   }
 
   public static final class id {
     @IdRes
-    public static final int res = 0x7f040008;
+    public static final int res = 8;
   }
 
   public static final class integer {
     @IntegerRes
-    public static final int res = 0x7f040009;
+    public static final int res = 9;
   }
 
   public static final class layout {
     @LayoutRes
-    public static final int res = 0x7f040010;
+    public static final int res = 10;
   }
 
   public static final class menu {
     @MenuRes
-    public static final int res = 0x7f040011;
+    public static final int res = 11;
   }
 
   public static final class plurals {
     @PluralsRes
-    public static final int res = 0x7f040012;
+    public static final int res = 12;
   }
 
   public static final class string {
     @StringRes
-    public static final int res = 0x7f040013;
+    public static final int res = 13;
   }
 
   public static final class style {
     @StyleRes
-    public static final int res = 0x7f040014;
+    public static final int res = 14;
   }
 
   public static final class styleable {
     @StyleableRes
-    public static final int resArray_child = 0;
+    public static final int resArray_child = 15;
```

Comment:
BTW, for children of declare styleables it's actually pretty interesting since, if we had:
<resources> 
    <attr name="foo"/>
    <declare-styleable name="bar">
        <item type="attr" name="foo"/> //re-using 'foo' defined above
    </declare-styleable>
</resoures>

In the (real) R class R.attr.foo and R.styleable.bar[R.styleable.bar_foo] should have the same value:
R.attr.foo = 12345
R.styleable.bar = [12345]
R.styleable.bar_foo = 0 // index in the R.styleable.bar array

I don't think this will affect this change at all, but just an FYI. :)

Reply:
Sorry, the comment got rid of all the sharp brackets... The code that was removed:
```
<resources>
    <attr name="foo"/>
    <declare-styleable name="bar">
        <item type="attr" name="foo"/> //re-using 'foo' defined above
    </declare-styleable>
</resources>
```